Formula/grass.rb

```diff
@@ -0,0 +1,352 @@
+      GRASS_ADDON_ETC=\"$addpath\"
+      fi
+      export GRASS_ADDON_ETC
+
+      mkdir -p \"$GISBASE_USER/Addons/lib\"
+
+      mkdir -p \"$GISBASE_USER/Addons/lib\"
+      mkdir -p \"$GISBASE_USER/Addons/docs/html\"
+
+      \"$app_dir/Contents/MacOS/etc/build_html_user_index.sh\" \"$GISBASE\"
+      \"$app_dir/Contents/MacOS/etc/build_gui_user_menu.sh\"
+
+      if [ ! \"$GRASS_FONT_CAP\" ] ; then
+      GRASS_FONT_CAP=\"$GISBASE_USER/Addons/etc/fontcap\"
+      fi
+      export GRASS_FONT_CAP
+
+      export GDAL_DATA=\"$GISBASE/share/gdal\"
+
+      # set Python
+      export GRASS_PYTHON=\"python3\"
+
+      # Add the GRASS start command
+      export cmd=\"$GRASS_PYTHON $GISBASE/grass --gui $@\"
+
+      # Use AppleScript to open a new terminal window and open GRASS
+      osascript -  \"$cmd\"  <<EOF
+      on run argv -- argv is a list of strings
+      tell application \"Terminal\"
+      if not (exists window 1) then reopen
+      activate
+      do script (item 1 of argv) in window 1
+      end tell
+      end run
+    EOS
+
+    File.write("#{grass_macos_app}/Contents/MacOS/grass.sh", file)
+    chmod("+x", "#{grass_macos_app}/Contents/MacOS/grass.sh")
+
+    config = <<~EOS
+      <?xml version="1.0" encoding="UTF-8"?>
+      <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+      <plist version="1.0">
+        <dict>
+      <key>CFBundleDevelopmentRegion</key>
+      <string>en</string>
+      <key>CFBundleExecutable</key>
+      <string>grass.sh</string>
+      <key>NSHumanReadableCopyright</key>
+      <string>Copyright © 1999–2022 GRASS Development Team</string>
+      <key>CFBundleIconFile</key>
+      <string>AppIcon</string>
+      <key>CFBundleName</key>
+      <string>GRASS-#{version}</string>
+      <key>CFBundleDisplayName</key>
+      <string>GRASS-#{version}</string>
+      <key>CFBundleIdentifier</key>
+      <string>org.osgeo.grass</string>
+      <key>CFBundleInfoDictionaryVersion</key>
+      <string>6.0</string>
+      <key>CFBundlePackageType</key>
+      <string>APPL</string>
+      <key>CFBundleShortVersionString</key>
+      <string>#{version}</string>
+      <key>CFBundleVersion</key>
+      <string>#{version}</string>
+      <key>CFBundleDocumentTypes</key>
+      <array>
+        <dict>
+          <key>CFBundleTypeName</key>
+          <string>GRASS Workspace File</string>
+          <key>CFBundleTypeExtensions</key>
+          <array>
+            <string>gxw</string>
+          </array>
+          <key>CFBundleTypeIconFile</key>
+          <string>GRASSDocument_gxw</string>
+        </dict>
+      </array>
+      <key>LSMinimumSystemVersion</key>
+      <string>12.3</string>
+      <key>LSHasLocalizedDisplayName</key>
+      <true/>
+        </dict>
+      </plist>
+    EOS
+
+    # File.open("#{grass_macos_app}/Contents/Info.plist", "w") { |file| file.write(config) }
+    File.write("#{grass_macos_app}/Contents/Info.plist", config)
+  end
```

Comment:
This is unmaintainable. If the `Makefile` doesn't produce this then we don't want it.